tutorials/visualization_tutorial.ipynb

```diff
@@ -0,0 +1,794 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    u = mda.Universe(datafiles.ea_fec_pdb, datafiles.ea_fec_dcd)
This PDB doesn't seem to exist as a datafile?
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/solvation-analysis/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Hmm, this is working locally for me. On GitHub I see <code>ea_fec.pdb</code> in the "files changed" and an <code>ea_fec_pdb</code>  variable added to <code>datafiles.py</code>. Are you having an issue running this cell?</p>